build/automation/var/project.mk

```diff
@@ -24,8 +24,8 @@ DEPLOYMENT_SECRETS = $(PROJECT_ID)-$(PROFILE)/deployment
 AWS_SAME_ACCOUNT_DOCKER_REGISTRY = $(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_DEFAULT_REGION).amazonaws.com/$(PROJECT_GROUP_SHORT)/$(PROJECT_NAME_SHORT)
 PYTHON_VERSION = $(PYTHON_VERSION_MAJOR).$(PYTHON_VERSION_MINOR)
```

Comment:
If I understand the issue correctly then this variable must be removed.

Reply:
You are correct, not sure how that crept back in, removing now ...